plugin-redfish/main.go

```diff
@@ -65,7 +65,7 @@ func main() {
 
 	configFilePath := os.Getenv("PLUGIN_CONFIG_FILE_PATH")
 	if configFilePath == "" {
-		log.Fatal("error: no value get the environment variable PLUGIN_CONFIG_FILE_PATH")
+		log.Fatal("No value get the environment variable PLUGIN_CONFIG_FILE_PATH")
```

Comment:
Please change it to "No value set for the environment variable PLUGIN_CONFIG_FILE_PATH"